app/i18n/en/gen.php

```diff
@@ -2,7 +2,7 @@
 
 return array(
 	'action' => array(
-		'actualize' => 'Actualize',
+		'actualize' => 'Actualise',
```

Comment:
Oxford's en_GB is *actualize* https://en.wiktionary.org/wiki/actualise but the -ise form is fine too in en_GB